src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -187,6 +193,25 @@ export class ViewMappingProfile extends Component {
     const tagsEntityLink = `data-import-profiles/mappingProfiles/${mappingProfile.id}`;
     const associations = [...[], ...mappingProfile.parentProfiles, ...mappingProfile.childProfiles];
 
+    const formConfig = formConfigSamples.find(cfg => cfg.name === formName);
+
+    const record = JSON.parse(JSON.stringify({ ...mappingProfile }));
```

Comment:
Because it's 100% way to wipe out objective links to a constant source.

Reply:
Can deepClone from lodash be used insted?

Reply:
No. _.cloneDeep copy is still shallow

Reply:
Could you please provide me an example of what you are trying to achieve.

Reply:
I think using the `cloneDeep` method from the lodash will be easier to understand.

Reply:
`FlexibleForm` modifies `record` object in static mode to augment it with calculated and placeholder values.
As far as the record (current profile data) is taken from `stripesConnect` state, it should be 100% independent when you copy it to use its own copy and leave `stripesConnect` state one immutable.
_.cloneDeep doesn't do it making deep but still a shallow copy.

Reply:
```
var objects = [{ 'a': 1 }, { 'b': 2 }];
 
var deep = _.cloneDeep(objects);
console.log(deep[0] === objects[0]);
// => false
```
Docs says otherwise. By the way there are others cloning methods in lodash which you can try to achive the needed behavior.

Reply:
Done